test/unit/math/fwd/mat/fun/resize_test.cpp

```diff
@@ -7,12 +7,12 @@ using stan::math::fvar;
 
 TEST(AgradFwdMatrixResize, fvar_double) {
   fvar<double> x = 5;
-  std::vector<size_t> dims;
+  std::vector<int> dims;
```

Comment:
In these tests I'm just changing size_t to int and changing how Vectors and RowVectors are resized.